src/applications/vaos/components/ExpressCareCard.jsx

```diff
@@ -6,16 +6,16 @@ import { getPatientPhone, getPatientEmail } from '../services/appointment';
 import { APPOINTMENT_STATUS } from '../utils/constants';
 import ExpressCareStatus from './ExpressCareStatus';
 
-export default function ExpressCareListItem({
+export default function ExpressCareCard({
```

Comment:
I needed to separate this out from being a list item, since it's used both in a list and outside of a list.